index.html

```diff
@@ -4039,8 +4039,34 @@ <h4><dfn>Fullscreen Window</dfn></h4>
   context</a>.
 </ol>
 </section> <!-- /Fullscreen Window -->
-
 </section> <!-- /Switch To Window -->
+<section>
+  <h3>Consume user activation of Window</h3>
+
+  <table class="simple jsoncommand">
+    <tr>
+    <th>HTTP Method</th>
+    <th>URI Template</th>
+    </tr>
+    <tr>
+    <td>POST</td>
+    <td>/session/{<var>session id</var>}/window/consume-user-activation</td>
+    </tr>
+  </table>
+
+  <p>The <a>remote end steps</a> are:
+
+  <ol>
+    <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
+    return <a>error</a> with <a>error code</a> <a>no such window</a>.
+
+    <li><p><a>Handle any user prompts</a> and return its value if it is an <a>error</a>.
+
+    <li><p><a>Consume user activation</a> of the <a>current browsing context</a>'s [=active window=].
```

Comment:
@domenic, [=active window=] is currently not exported from HTML... should it be? Seems pretty useful. Or should I use something else.

Reply:
Eh, we kind of want to discourage people from using browsing contexts, but seeing as how this whole spec is currently based on them, I guess we should... it'll be `[=browsing context/active window=]` though, to distinguish it from the (more generally recommended) `[=navigable/active window=]`.

Reply:
Actually, I'd prefer we keep it un-exported for now, and require this spec to manually add it in the anchors block since it's doing something ill-advised.